src/components/admin/settings-documents-table.js

```diff
@@ -58,8 +58,8 @@ function SettingsDocumentsTable({ settingsDocuments, onChange }) {
 
   const renderRank = (text, record, index) => (
     <span style={{ whiteSpace: 'nowrap' }}>
-      <Button size="small" icon={<MoveUpIcon />} disabled={index === 0} onClick={() => handleMoveClick(index, -1)} />
-      <Button size="small" icon={<MoveDownIcon />} disabled={index === settingsDocuments.length - 1} onClick={() => handleMoveClick(index, +1)} />
+      <Button style={{ margin: '0 3px' }} size="small" icon={<MoveUpIcon />} disabled={index === 0} onClick={() => handleMoveClick(index, -1)} />
+      <Button style={{ margin: '0 3px' }} size="small" icon={<MoveDownIcon />} disabled={index === settingsDocuments.length - 1} onClick={() => handleMoveClick(index, +1)} />
```

Comment:
Maybe we could have a utility class here like `u-small-horizontal-margin` instead of the inline styles? (or one on the parent like `u-small-button-group` that spaces the children?)

Reply:
Yes, much better, did that!

Reply:
Poifect!